mod_ci/controllers.py

```diff
@@ -1160,16 +1160,6 @@ def progress_type_request(log, test, test_id, request) -> bool:
                 time_diff = (prep_finish_time - kvm_entry.timestamp).total_seconds()
```

Comment:
I would like to discuss two points here:
1. Have we changed the enum `TestStatus` to not have testing as a value?
2. How is `kvm_entry` valid here?